modules/apps/headless/headless-object/headless-object-impl/src/main/java/com/liferay/headless/object/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java

```diff
@@ -281,7 +281,11 @@ public ObjectEntryFolder postScopeScopeKeyObjectEntryFolder(
 					contextAcceptLanguage.getPreferredLocale(),
 					objectEntryFolder.getLabel(),
 					objectEntryFolder.getLabel_i18n()),
-				objectEntryFolder.getName()));
+				objectEntryFolder.getName(),
+				ServiceContextBuilder.create(
+					persistedObjectEntryFolder.getGroupId(),
```

Comment:
@Mikellorza, I know we have the persistedObjectEntryFolder, but can't we use the groupId variable that we have on line 258 here?

Reply:
done!